pkg/driver/driver.go

```diff
@@ -1087,6 +1092,11 @@ func (d Driver) ControllerExpandVolume(ctx context.Context, req *csi.ControllerE
 	}, nil
 }
 
+// ControllerGetVolume https://github.com/container-storage-interface/spec/blob/v1.4.0/spec.md#controllergetvolume
+func (d Driver) ControllerGetVolume(ctx context.Context, request *csi.ControllerGetVolumeRequest) (*csi.ControllerGetVolumeResponse, error) {
+	panic("implement me")
```

Comment:
that really is intentional?

Reply:
You are right, better to respond with a proper `Unimplemented` error